src/puppetlabs/puppetdb/http/events.clj

```diff
@@ -3,10 +3,11 @@
   (:require [puppetlabs.kitchensink.core :as kitchensink]
             [puppetlabs.puppetdb.query.paging :as paging]
             [puppetlabs.puppetdb.query-eng :refer [produce-streaming-body]]
-            [clj-time.coerce :refer [to-timestamp]]
             [puppetlabs.puppetdb.middleware :as middleware]
             [net.cgrand.moustache :refer [app]]
-            [puppetlabs.puppetdb.http :as http]))
+            [puppetlabs.puppetdb.http :as http]
+            [puppetlabs.puppetdb.utils :as utils]
```

Comment:
Yes, I had that to-timestamp fn in utils, then moved it. This like is no longer needed.